tests/main/parallel-install-auto-aliases/task.yaml

```diff
@@ -73,14 +69,21 @@ execute: |
     # clean slate
     snap remove --purge test-snapd-auto-aliases
 
-    echo "When test-snapd-auto-aliases_foo is installed"
+    echo "Even when test-snapd-auto-aliases_foo is installed first"
+    echo "it should be installed with --unaliased"
     snap install test-snapd-auto-aliases_foo
 
-    echo "Installing test-snapd-auto-aliases will conflict"
-    # TODO parallel-install: need to install using file
-    not snap install "$name"
-    snap change --last=install | MATCH 'cannot enable aliases .* for "test-snapd-auto-aliases", already enabled for "test-snapd-auto-aliases_foo"'
+    echo "Installing test-snapd-auto-aliases after test-snapd-auto-aliases_foo"
+    echo "shouldn't conflict and it's aliases should be enabled"
+    snap install test-snapd-auto-aliases
```

Comment:
I guess this check is time dependent right? If it was failing, it's possible that we wouldn't know before the first install finished before we started this one

Reply:
Oh sorry never mind, I misread the test